tinker/templates/profile.html

```diff
@@ -0,0 +1,61 @@
+{% extends 'tinker_base.html' %}
+
+{% set title = 'Profile' %}
+
+{% block styles %}
+    <style type="text/css">
+        h2 {
```

Comment:
It doesn't look like you are using li, so that could be removed. 

And the th/td, could you apply the classes to a single table? I wouldn't spend too much time on this, as this page isn't too big of a deal. But it is better code practice, so some change is probably needed.